tests/foreman/cli/test_http_proxy.py

```diff
@@ -264,14 +264,13 @@ def test_positive_assign_http_proxy_to_products(module_org, module_target_sat):
     )
 
     # Set the HTTP proxy through bulk action for both products to the selected proxy
-    res = module_target_sat.cli.Product.update_proxy(
+    module_target_sat.cli.Product.update_proxy(
```

Comment:
Just curious, what stdout do we get now when running product update_proxy in CLI?

Reply:
It looks to me like stdout for the update_proxy CLI command is now the progress bar shown in the problem statement above.

Reply:
IIRC the `res.stdout` was `None` (not even the progress bar).